change/@microsoft-teams-js-889e1105-9a8b-468a-94d7-ed8253cdf569.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Made ApiName enum const to reduce package size",
```

Might make this more generic; since the ApiName enum isn't exported for customers this is maybe more detail than they need.

```suggestion
  "comment": "Made enum const to reduce package size",
```